.eslintignore

```diff
@@ -0,0 +1,2 @@
+app/javascript/packs
+app/javascript/channels
```

Comment:
Out of curiosity, why have you chosen to exclude these folders from the eslint run?

Reply:
When running `npx eslint app/javascript/`, they were appearing with errors for library content.

```
/Users/megan.duffy/WPE/OnboardingProject/app/javascript/channels/index.js
  4:18  error  'require' is not defined  no-undef

/Users/megan.duffy/WPE/OnboardingProject/app/javascript/packs/application.js
   6:1   error  'require' is not defined  no-undef
   7:1   error  'require' is not defined  no-undef
   8:1   error  'require' is not defined  no-undef
   9:1   error  'require' is not defined  no-undef
  19:31  error  'require' is not defined  no-undef
  20:21  error  'require' is not defined  no-undef

/Users/megan.duffy/WPE/OnboardingProject/app/javascript/packs/server_rendering.js
  3:31  error  'require' is not defined  no-undef
  4:21  error  'require' is not defined  no-undef
```

As I don't have any custom code in there, it makes sense for me to ignore them for the time being. Thoughts?

Reply:
This questions might help. Look at the answer with the most up votes https://stackoverflow.com/questions/57673026/require-and-process-is-not-defined-in-eslint-problem-with-node

Reply:
Did the trick nicely :)